docs/CONTRIBUTING.md

```diff
@@ -80,19 +79,19 @@ We try to have at least two people review every pull request and we encourage ev
 - Is the code easy to understand and to maintain?
 - Is there sufficient detail to inform any changes to documentation?
 
-When a pull request is first created, @getodk-bot tags it as `needs review` to indicate that code review is needed. Community members review the code and leave their comments, verifying that the changes included are relevant and properly address the issue. A maintainer does a thorough code review and when satisfied with the code, tags the pull request as `needs testing` to indicate the need for a manual [black-box testing](https://en.wikipedia.org/wiki/Black-box_testing) pass. A pull request may go back and forth between `needs testing` and `needs review` until the behavior is thoroughly verified. Once the behavior has been thoroughly verified, the pull request is tagged as `behavior verified`. A maintainer then merges the changes. Pull requests that need more complete reviews including review of approach and/or appropriateness are tagged with `reviews wanted`. Any community member is encouraged to participate in the review process!
+Community members review the code and leave their comments, verifying that the changes included are relevant and properly address the issue. A maintainer does a thorough code review and when satisfied with the code, accepts and may label the pull request as `needs testing` to indicate the need for a manual [black-box testing](https://en.wikipedia.org/wiki/Black-box_testing) pass. A pull request may have `needs testing` removed and added again several times until the behavior is thoroughly verified. Once the behavior has been verified, the pull request is labelled as `behavior verified`.
 
-Small fixes that target very particular bugs may occasionally be merged without a second review.
+## Merging pull requests
 
-## Work in progress pull requests
+Pull requests can be merged by committers once they are accepted or once they have been accepted and have been labelled as `behavior verified` unless:
```

Comment:
Added this to make it explicit that CI has to be green for us to merge. Also thought it was good to specify that only committers can merge and get a mention of Code Freeze in here (currently only in `README.md`).